charts/scalardb/ci/scalardb-ct-values.yaml

```diff
@@ -0,0 +1,6 @@
+scalardb:
+  databaseProperties: |
+    scalar.db.storage=jdbc
+    scalar.db.contact_points=jdbc:postgresql://postgresql.default.svc.cluster.local:5432/postgres
+    scalar.db.username=postgres
+    scalar.db.password=postgres
```

Comment:
The `ct` command automatically uses `${ROOT_DIR_OF_CHART}/ci/*-values.yaml` files as custom values files when it test each chart. 
https://github.com/helm/chart-testing/blob/main/doc/ct_install.md

In this update, we have to configure `database.properties` to access PostgreSQL.